modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/CartResourceImpl.java

```diff
@@ -257,6 +257,61 @@ private long _getCommerceRegionId(
 		return commerceRegion.getCommerceRegionId();
 	}
 
+	private CartItem[] _getValidatedCommerceOrderItems(
+			CommerceOrder commerceOrder)
+		throws Exception {
+
+		Map<Long, List<CommerceOrderValidatorResult>>
+			commerceOrderValidatorResults =
+				_commerceOrderValidatorRegistry.
+					getCommerceOrderValidatorResults(null, commerceOrder);
+
+		List<CommerceOrderItem> commerceOrderItems =
+			commerceOrder.getCommerceOrderItems();
+
+		List<CartItem> cartItems = new ArrayList<>();
+
```

Comment:
Lets sort cartItems to the top since everything else should be wrapped inside it.

```suggestion
		List<CartItem> cartItems = new ArrayList<>();

		Map<Long, List<CommerceOrderValidatorResult>>
			commerceOrderValidatorResults =
				_commerceOrderValidatorRegistry.
					getCommerceOrderValidatorResults(null, commerceOrder);

		List<CommerceOrderItem> commerceOrderItems =
			commerceOrder.getCommerceOrderItems();

```